sound/soc/sof/mediatek/Kconfig

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+
+config SND_SOC_SOF_MTK_TOPLEVEL
+	bool "SOF support for MTK audio DSPs"
+	depends on ARM64|| COMPILE_TEST
```

Comment:
Missing space, should be "ARM64 || COMPILE_TEST"

Reply:
ok. I will modify it, thanks